src/lavinmq/config.cr

```diff
@@ -41,6 +41,8 @@ module LavinMQ
     property? guest_only_loopback : Bool = true
     property max_message_size = 128 * 1024**2
     property? log_exchange : Bool = false
+    property free_disk_min : Int64 = 0  # bytes
+    property free_disk_warn : Int64 = 0 # bytes
```

Comment:
Should we consider some defaults here?

Reply:
Skipping defaults here since lavinmq can be run on a lot of different hardware. I'll add some notes to the documentation about it instead.
We could also add some default values to be set for CloudAMQP deploys.